src/tests/ftest/io/small_file_count.yaml

```diff
@@ -22,7 +22,7 @@ server_config:
       storage: auto
 pool:
   scm_size: 40G
-  nvme_size: 350G
+  nvme_size: 300G
```

Comment:
I'm not sure the intention of this test, but maybe `size: 95%` would work

Reply:
Yup, that could be better ideally, but now we have known timeout issues with ratio > 90% which is still in-progress fixing, i'd like to land this as it for now.